nixos/maintainers/scripts/ec2/amazon-image-zfs.nix

```diff
@@ -0,0 +1,25 @@
+{
+
+  imports = [ ./amazon-image.nix ];
+  # ec2.efi = true;
+  ec2.zfsRoot = true;
+  networking.hostId = "00000000";
+  fileSystems = {
+    "/" = {
+      device  = "tank/system/root";
+      fsType = "zfs";
+    };
+    "/home" = {
+      "device" = "tank/user/home";
+      "fsType" = "zfs";
+    };
+    "/nix" = {
+      "device" = "tank/local/nix";
+      "fsType" = "zfs";
+    };
+    "/var" = {
+      "device" = "tank/system/var";
+      "fsType" = "zfs";
+    };
+  };
```

Comment:
This is a bit weird, it is embedding this config with the idea that it matches `make-zfs-image`'s default datasets:

```

  datasets ? {
    "system/root".mount = "/";
    "system/var".mount = "/var";
    "local/nix".mount = "/nix";
    "user/home".mount = "/home";
  }
```